paasta_tools/firewall.py

```diff
@@ -0,0 +1,117 @@
+from __future__ import absolute_import
+from __future__ import unicode_literals
+
+import collections
+import hashlib
+
+from paasta_tools import iptables
+
+
+PRIVATE_IP_RANGES = (
```

Comment:
There probably are packages out there where these constants are defined, but I prefer what you've done here and made it explicit and under our control. ++